modules/op-creating-pipeline-tasks.adoc

```diff
@@ -0,0 +1,61 @@
+// Ths module is included in the following assembly:
+//
+// *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc
+
+[id="creating-pipeline-tasks_{context}"]
+= Creating Pipeline Tasks
+
+[discrete]
+.Procedure
+
+. Install the `apply-manifests` and `update-deployment` Tasks from the `pipelines-tutorial` repository, which contains a list of reusable Tasks for Pipelines:
++
+----
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/release-tech-preview-1/01_pipeline/01_apply_manifest_task.yaml
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/release-tech-preview-1/01_pipeline/02_update_deployment_task.yaml
+----
+
+. Verify the Tasks you created:
++
+----
+$ tkn task list
+
+NAME                DESCRIPTION   AGE
+apply-manifests                   1 minute ago
+update-deployment                 48 seconds ago
+----
+
+. Verify the Operator installed additional ClusterTasks, for example --`buildah` and `s2i-python-3`:
++
+----
+$ tkn clustertasks list
+NAME                       DESCRIPTION   AGE
+buildah                                  1 day ago
+buildah-v0-11-3                          1 day ago
```

Comment:
add this after buildah-v0-11-3
 
git-clone                                1 day ago